_content/tour/rus/algorithms-fun.article

```diff
@@ -86,46 +83,41 @@ length quantities that they translate to:
     08000000          C0 80 80 00
     0FFFFFFF          FF FF FF 7F
 
-A variable-length quantity (VLQ) is a universal code that uses an arbitrary
-number of binary octets (eight-bit bytes) to represent an arbitrarily large
-integer. It was defined for use in the standard MIDI file format[1] to save
-additional space for a resource constrained system, and is also used in the
-later Extensible Music Format (XMF). A VLQ is essentially a base-128
-representation of an unsigned integer with the addition of the eighth bit
-to mark continuation of bytes. See the example below.
+Код переменной длины (`VLQ`) - это универсальный код, использующий произвольное количество
+двоичных октетов (восьмибитных байтов) для представления произвольно большого
+целого числа. Он был создан для использования в стандартном формате MIDI-файлов[1], чтобы сэкономить
+дополнительное пространство для системы с ограниченными ресурсами, а также используется в более позднем формате
+`Extensible Music Format (XMF)`. VLQ - это, по сути, base-128
+представление целого беззнакового числа с добавлением восьмого бита
+для обозначения продолжения. Смотрите пример ниже.
 
 	Int:    16384
 	IntHex: 0x00004000
 	IntBin: 00000000 00000000 01000000 00000000
 	VLQHex: 0x81 0x80 0x00
 	VLQBin: 00000000 10000001 10000000 00000000
 
-Lets say I want to represent the number 3435 in VLQ. 3435 in
-binary is 110101101011. We can not fit this in a byte. So we will
-chop it up from the end in 7-bit blocks.
+Допустим, мы хотим представить число 3435 в `VLQ`. `3435` в
+двоичном виде - это `110101101011`. Мы не можем уместить это число в байте. Поэтому мы
+будем разбивать его с конца на 7-битные блоки.
 
     Septet	7	6	5	4	3	2	1
     #1		1	1	0	1	0	1	1
     #2		0	0	1	1	0	1	0
 
-Now we prepend all but the last with a 1-bit to indicate that an octet
-follows and prepend a 0-bit to the last, signalling the final octet.
+Теперь перед всеми октетами, кроме последнего, мы ставим 1 бит, чтобы указать, что за ним следует октет
+и добавляем 0-бит к последнему, сигнализируя о последнем октете.
 
     Octet   8	7	6	5	4	3	2	1
     #1      0	1	1	0	1	0	1	1
     #2      1	0	0	1	1	0	1	0
 
-Finally we concatenate them, most significant octet first, into
+Наконец, мы объединяем их, старший октет, в
 
 Encoded: 10011010 01101011  ToHex: 0x9A 0x6B
 
-*Extra* *Resources:*
```

Comment:
Is there a particular reason why this section is excluded from the translation?

Reply:
no, no reason at all